src/logging/logger.ts

```diff
@@ -36,6 +36,8 @@ export class ComponentLogger implements ZLUX.ComponentLogger {
   private parentLogger:Logger;
   private componentName:string;
   private _messages: MessageTable;
+  //for keeping them in sync
+  private _subLoggers: ComponentLogger[] = [];
```

Comment:
For consistency's sake, we should probably initialize this with the rest of the class objects in the constructor

Reply:
Sure